src/Microsoft.Health.Dicom.Functions/Indexing/ReindexFunction.cs

```diff
@@ -121,7 +121,7 @@ public static Task ProcessBatchAsync([ActivityTrigger] ReindexBatch batch, ILogg
             return Task.CompletedTask;
         }
 
-        [FunctionName("ReindexV1_Timer")]
+        //[FunctionName("ReindexV1_Timer")]
```

Comment:
remove this change?

Reply:
I'll keep it for now as this timer may interfere with local debugging and is probably unnecessary in the end